src/System.Management.Automation/namespaces/LocationGlobber.cs

```diff
@@ -4564,20 +4516,16 @@ internal static bool IsHomePath(string path)
         /// <exception cref="ArgumentNullException">
         /// If <paramref name="path"/> is null.
         /// </exception>
-        internal static bool IsProviderDirectPath(string path)
+        internal static bool IsProviderDirectPath(ReadOnlySpan<char> path)
         {
-            if (path == null)
-            {
-                throw PSTraceSource.NewArgumentNullException(nameof(path));
-            }
-
```

Comment:
Here as well with the exception doc comment.

Reply:
Done.